### PR TITLE
Refac blog

### PR DESCRIPTION
**Pull request by Cellou404**
Refactoring code

**Screenshoot**

Capture d’écran du 2024-09-01 10-58-12
Capture d’écran du 2024-09-01 10-59-02

### CODE REVIEW

File: blog/serializers/__init__.py
Comment (CellouCS) [inline blog/serializers/__init__.py:1]: thanks for adding this file

File: blog/serializers/categories.py
Comment (CellouCS) [inline blog/serializers/categories.py:1]: Good approch

File: blog/serializers/posts.py
Comment (CellouCS) [inline blog/serializers/posts.py:1]: Well organizerd

**Review by CellouCS — APPROVED** [review]
Approved

File: blog/serializers/__init__.py
Comment (Cellou404) [reply blog/serializers/__init__.py:1]: it was necessary

File: blog/serializers/categories.py
Comment (Cellou404) [reply blog/serializers/categories.py:1]: 👍🏽

File: blog/serializers/posts.py
Comment (Cellou404) [reply blog/serializers/posts.py:1]: 👍🏽

**Pull request merged**